### PR TITLE
[W5][W09-1] Ng Wei Xin

### PR DESCRIPTION
**Pull request by NgWeiXin**

### CODE REVIEW

**Comment by shimeilee** [comment]
The UndoDeleteCommand only removes the last deleted person. Could consider creating a stack to store the last few deleted persons.

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -111,6 +111,16 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+== Undo-ing previous delete: `undoDelete`
+
+Undo the latest delete command.
+Format: `undoDelete`
```
Comment (okkhoy) [inline docs/UserGuide.adoc:117]: Good job updating the user guide :+1:

File: src/seedu/addressbook/commands/UndoDeleteCommand.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+public class UndoDeleteCommand extends Command {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/UndoDeleteCommand.java:7]: Classes and non trivial methods need header comments.

**Review by okkhoy — COMMENTED** [review]
Ovrall a useful enhancement.
Do remember to update the tests for all new features added and add header comments everywhere necessary.

**Comment by okkhoy** [comment]
@NgWeiXin 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**